ParentReact/src/Rob.tsx

```diff
@@ -0,0 +1,11 @@
+import { useSvelteStore } from "./useSvelteStore";
+
+export const Rob = () => {
+  const { count, randomizeValue } = useSvelteStore();
+
+  return (
+    <>
+      <div>From rob: {count}</div>
+      <button onClick={randomizeValue}>Randomize rob</button>
+    </>)
+}
```

Comment:
I think this simplicity is what we're going for; The underlying lib takes on the complexity to provide a friendly interface